manual/core/shaded_jar/README.md

```diff
@@ -0,0 +1,39 @@
+## Using the shaded JAR
+
+The default driver JAR depends on [Netty](http://netty.io/), which is
+used internally for networking.
+
+This explicit dependency can be a problem if your application already
+uses another Netty version. To avoid conflicts, we provide a "shaded"
+version of the JAR, which bundles the Netty classes under a different
+package name:
+
+```xml
+<dependency>
+  <groupId>com.datastax.oss</groupId>
+  <artifactId>java-driver-core-shaded</artifactId>
+  <version>4.0.0-alpha3</version>
```

Comment:
Since it's a completely different artifact, we don't need to exclude netty as a dependency anymore which is nice.